data_steward/deid/aou.py

```diff
@@ -335,7 +334,7 @@ def map_questionnaire_response_ids(self, lower_bound):
 
         :param lower_bound:  The smallest number that may be used as an identifier.
         """
-        map_tablename = "_deid_questionnaire_response_map"
+        map_tablename = self.idataset + "._deid_questionnaire_response_map"
```

Comment:
Interesting that this failed.  It was supposed to use the dataset name passed by the config dictionary.  This same type of query and config is used in other places in aou.py.  I wonder why this one failed.